nc-games/src/utils/api.js

```diff
@@ -9,3 +9,9 @@ export const getReviews = () => {
     return data.reviews;
   });
 };
+
+export const getReviewById = (review_id) => {
+  return ncGamesApi.get(`/reviews/${review_id}`).then(({ data }) => {
```

Comment:
Great use of the API here to get the full review details instead of filtering the reviews array 💯